sql/core/src/test/scala/org/apache/spark/sql/execution/command/v1/DescribeTableSuite.scala

```diff
@@ -225,4 +246,30 @@ class DescribeTableSuite extends DescribeTableSuiteBase with CommandSuiteBase {
           Row("Partition Provider", "Catalog", "")))
     }
   }
+
+  test("DESCRIBE TABLE EXTENDED of a table with a default column value") {
```

Comment:
nit: shall we update `describe.sql` so that we can have a straightforward end-to-end test?

Reply:
yes, I have added this testing as well.